src/index.html

```diff
@@ -17,16 +17,33 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="form"
+      action="#"
+      data-qa="big"
+      method="get"
+    >
+      <input
+        class="search search--big"
+        name="search-big"
+        data-qa="keypress"
+        type="text"
+        placeholder="Try “Los Angeles”"
+      />
+    </form>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="form"
+      action="#"
+      method="get"
+      data-qa="small"
+    >
+      <input
+        class="search search--small"
+        name="search-small"
+        type="text"
+        placeholder="Try “Los Angeles”"
+      />
```

Comment:
Similar to the previous comment, this `form` element also lacks `label` elements. Remember that accessibility is an important aspect of web development. Even invisible labels can greatly improve the experience for users of screen readers.